plugins/modules/sns_topic.py

```diff
@@ -614,6 +648,7 @@ def main():
         purge_subscriptions=dict(type='bool', default=True),
         tags=dict(type='dict', aliases=['resource_tags']),
         purge_tags=dict(type='bool', default=True),
+        content_based_deduplication=dict(choices=['enabled', 'disabled'])
```

Comment:
Should't we also specify `default='disabled'`?

Reply:
No, that forces an overwrite if it's not set and potentially breaks existing topics